cpp/src/replace/nans.cu

```diff
@@ -26,6 +26,7 @@
 #include <cudf/utilities/type_dispatcher.hpp>
 
 #include <rmm/cuda_stream_view.hpp>
+#include <rmm/exec_policy.hpp>
```

Comment:
I'm curious why this change was necessary.

Reply:
This was previously included in the include file `copy_if_else.cuh` which did not use it. Cleaning up the headers there required moving it here where it is actually used.